Oracle/PCORNetLoader_ora.sql

```diff
@@ -415,66 +415,6 @@ begin
 end;
 /
 
-BEGIN

```

Comment:
@dckc, yes, it seemed like it was best to create/populate the PMN_LabNormal table from a .csv rather than doing a bunch of inserts with hard-coded strings.  As I recall, the motivation was that Mike needed access to those values earlier in the process.

We hoped this change would be something that SCILHS would merge back to their version.